Tests/SonarScanner.MSBuild.PreProcessor.Test/PreProcessorTests.cs

```diff
@@ -353,88 +237,46 @@ public async Task PreProc_NoProject()
-
+            // Checks that local settings are used when creating the SonarLint.xml file, overriding
+            using var scope = new TestScope(TestContext);
+            var factory = new MockObjectFactory();
+            factory.Server.Data.ServerProperties.Add("shared.key1", "server shared value 1");
+            factory.Server.Data.ServerProperties.Add("shared.CASING", "server upper case value");
             // Local settings that should override matching server settings
             var args = new List<string>(CreateArgs())
             {
                 "/d:local.key=local value 1",
                 "/d:shared.key1=local shared value 1 - should override server value",
                 "/d:shared.casing=local lower case value"
             };
+            var settings = factory.ReadSettings();
+            var preProcessor = new PreProcessor(factory, factory.Logger);
 
-            var mockAnalyzerProvider = MockAnalyzerProvider();
-            var mockTargetsInstaller = new Mock<ITargetsInstaller>();
-            var mockFactory = new MockObjectFactory(mockServer, mockTargetsInstaller.Object, mockAnalyzerProvider);
-
-            TeamBuildSettings settings;
-            using (PreprocessTestUtils.CreateValidNonTeamBuildScope())
-            using (new WorkingDirectoryScope(workingDir))
-            {
-                settings = TeamBuildSettings.GetSettingsFromEnvironment(new TestLogger());
-                settings.Should().NotBeNull("Test setup error: TFS environment variables have not been set correctly");
-                settings.BuildEnvironment.Should().Be(BuildEnvironment.NotTeamBuild, "Test setup error: build environment was not set correctly");
-
-                var preProcessor = new PreProcessor(mockFactory, logger);
-
-                // Act
-                var success = await preProcessor.Execute(args);
-                success.Should().BeTrue("Expecting the pre-processing to complete successfully");
-            }
-
-            // Assert
-
-            // Check the settings used when creating the SonarLint file - local and server settings should be merged
-            mockAnalyzerProvider.SuppliedSonarProperties.Should().NotBeNull();
-            mockAnalyzerProvider.SuppliedSonarProperties.AssertExpectedPropertyValue("server.key", "server value 1");
-            mockAnalyzerProvider.SuppliedSonarProperties.AssertExpectedPropertyValue("local.key", "local value 1");
-            mockAnalyzerProvider.SuppliedSonarProperties.AssertExpectedPropertyValue("shared.key1", "local shared value 1 - should override server value");
-            // Keys are case-sensitive so differently cased values should be preserved
-            mockAnalyzerProvider.SuppliedSonarProperties.AssertExpectedPropertyValue("shared.CASING", "server upper case value");
-            mockAnalyzerProvider.SuppliedSonarProperties.AssertExpectedPropertyValue("shared.casing", "local lower case value");
```

Comment:
These checks are not needed anymore?

Reply:
Good catch. That doesn't seem intentional, maybe a rebase issue